### PR TITLE
Improve E2E test run and coverage

### PR DESCRIPTION
**Pull request by lachieh**
- **test(ts-e2e): fix e2e runner in CI**

### CODE REVIEW

**Review by joonas — APPROVED** [review]

File: typescript/apps/washboard-ui/src/components/data-table/index.tsx [hunk 1]
```diff
@@ -57,7 +66,7 @@ export function DataTable<TData, TValue>({
           <TableBody>
             {table.getRowModel().rows?.length ? (
               table.getRowModel().rows.map((row) => (
-                <TableRow key={row.id}>
+                <TableRow key={row.id} data-testid="" {...generateRowProps(row.original)}>
```
Comment (vados-cosmonic) [inline typescript/apps/washboard-ui/src/components/data-table/index.tsx]: should this the testid here be empty?

**Review by vados-cosmonic — APPROVED** [review]
The new test seems to still be failing/reporting a failure, but approving for other than that the code looks good to me.

File: typescript/apps/washboard-ui/src/components/data-table/index.tsx (hunk 1 above)
Comment (lachieh) [reply typescript/apps/washboard-ui/src/components/data-table/index.tsx]: Pulled this PR back to just the necessary CI changes but, yes, I'll fix this in the next PR.

Comment (lachieh) [reply typescript/apps/washboard-ui/src/components/data-table/index.tsx]: Follow up added in #3796

**Pull request merged**